cookbooks/aws-parallelcluster-slurm/recipes/config/login_nodes_daemon_service.rb

```diff
@@ -0,0 +1,54 @@
+require 'yaml'
+config = YAML.safe_load(File.read(node['cluster']['cluster_config_path']))
+GRACETIME_PERIOD = config["LoginNodes"]["Pools"][0]["GRACETIME_PERIOD"]
+
+# Create configuration file
+template "#{node['cluster']['shared_dir']}/login_nodes_daemon_config.json" do
+  source 'login_nodes_daemon_service/login_nodes_daemon_config.json.erb'
+  owner 'root'
+  group 'root'
+  mode '0644'
+  variables(
```

Comment:
variables accepts key arguments, not a dictionary. So, this should be:

```
variables(
  GRACETIME_PERIOD: ....
)
```

Reply:
Thank you, I have corrected it.